### PR TITLE
Github issue templates for acceptance testing, SAS integration and PGE release tasks

### LINKED ISSUE #519: [New Feature]: Develop issue templates for PGE Acceptance Testing and Integration
### Checked for duplicates

Yes - I've already checked

### Alternatives considered

Yes - and alternatives don't suffice

### Related problems

@RKuttruff had a great suggestion to develop some issue templates for our PGE Acceptance Testing and Integration that rolls up the common tasks we perform for each PGE (product comparison, wiki page creation etc) into standard task lists. This should hopefully make the processes clearer for present and future developers going forward.

### Describe the feature request

Create two new github issue templates: one for Acceptance Testing tasks, and one for PGE/SAS Integration tasks.

The previous tickets for these issue types should be consulted to determine what the common tasks are that we can abstract into a generic task list.

### PR DESCRIPTION
**Pull request by collinss-jpl**
## Description
- This branch adds additional issue template types for the following common activites:
   - SAS Delivery Acceptance Testing
   - PGE/SAS Integration
   - PGE Release

Unfortunately I don't think we can preview what these will look like rendered until this branch is merged, so for this review try to focus on information content and once its merged we can make further refinements if some of the formatting is off.

## Affected Issues
- Resolves #519 

## Testing
- N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/acceptance_test.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: SAS Acceptance Test
+description: Acceptance Testing Procedure for SAS Deliveries
+title: "<SAS Name> <Delivery> SAS <Version> Acceptance Test"
+labels: ["must have"]
+body:
+  - type: textarea
```
Comment (RKuttruff) [inline .github/ISSUE_TEMPLATE/acceptance_test.yml:6]: I think these might need to be changed to markdown types. You can preview how the template will render by viewing the file. ( ... -> View File)

image

Comment (RKuttruff) [reply .github/ISSUE_TEMPLATE/acceptance_test.yml:6]: I think I may have misunderstood what is being rendered here and the text area is definitely what we want.

Comment (collinss-jpl) [reply .github/ISSUE_TEMPLATE/acceptance_test.yml:6]: Didn't know about the View File feature, but it looks like that preview is rendered differently than how github itself would display the issue form, so it looks like we'll still just have to wait till branch is merged

File: .github/ISSUE_TEMPLATE/acceptance_test.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          - [ ] Optional: Download the delivered documentation for reference
+             - <Placeholder for Artifactory path to documentation>
+          - [ ] Load SAS docker image and deploy test data assets on development machine
+          - [ ] Modify provided RunConfig(s) as necessary (see previous AT wiki instructions, if available)
+          - [ ] Execute SAS container using provided sample inputs and RunConfig
+          - [ ] Perform Quality Assurance comparison check of output products with expected products from "golden" test data
+          - [ ] Create/Update Acceptance Testing Procedure wiki page
+             - Be sure to include any changes to test data structure, file permissions, RunConfig changes etc.
+          - [ ] Include the results of Quality Assurance check at the bottom of the wiki page
+             - Include any details on known failures or workarounds, and include a sample of the expected output from the comparison script
```
[the remark this answers is not in the document]
Comment (collinss-jpl) [reply .github/ISSUE_TEMPLATE/acceptance_test.yml:25]: 👍

**Review by RKuttruff — APPROVED** [review]
Looks good!

**Pull request merged**